Cargo.toml

```diff
@@ -14,5 +14,5 @@ lto = "fat"
 codegen-units = 1
 
 [dependencies]
-polkavm-derive = { version = "0.17.0" }
+polkavm-derive = { version = "0.19.0" }
 uapi = { package = "pallet-revive-uapi", git = "https://github.com/paritytech/polkadot-sdk.git", rev = "3d8da815ecd12b8f04daf87d6ffba5ec4a181806", default-features = false }
```

Comment:
Also needs to be bumped to what the latest revive release is using:
https://github.com/paritytech/revive/blob/fe1b3258d2956e51e2edd86f2e77898e6b142729/Cargo.toml#L76